### ISSUE TITLE
onnx2ncnn 不支持 ReduceL2, ReduceMax 之类的op

### ISSUE THREAD

**Issue by daichuanliang** [Tencent/ncnn#1306]
1. pytorch 的模型会用到一些torch.norm, torch,max 之类的操作，想转成ncnn。 先export成onnx后，再用onnx2ncnn 发现ReduceL2不支持

**Comment by nihui** [comment]
是不是 torch.nn.functional.normalize ？

**Comment by daichuanliang** [comment]
> 是不是 torch.nn.functional.normalize ？

不是，是torch.norm()
具体如下：
1.我想将pytorch模型转为（pytorch--->onnx--->onnx2ncnn)
2.其中 pytorch中torch.norm ---> onnx中的ReduceL2, 然后onnx2ncnn无法识别这个op

**Comment by bindog** [comment]
我目前正在尝试增加这些op支持

**Comment by daichuanliang** [comment]
> 我目前正在尝试增加这些op支持

您好，请问大概多久可以支持？ 增加后会commit到ncnn master吗？

**Comment by bindog** [comment]
几天之内吧，写好之后会提PR的

**Comment by bindog** [comment]
@daichuanliang 你着急用的话可以先clone我的ncnn/dev分支，编译出来试试看
功能上还有许多待完善的细节，我还要再花点时间

**Comment by nihui** [comment]
merged in https://github.com/Tencent/ncnn/commit/04b4b023249865ae01befe4170a873d5f2ec33f8

**Comment by MankaranSingh** [comment]
这些会与 vulkan 加速一起工作吗？

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
